app/build.gradle

```diff
@@ -110,6 +110,8 @@ dependencies {
     androidTestImplementation(libraries.expressoCore, {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
+
+    implementation 'com.android.support:customtabs:27.0.2'
```

Comment:
Next time please put the dependency on dependencies.gradle file! :)
I'll do this later.